db/schema.rb

```diff
@@ -50,6 +50,16 @@
     t.index ['schedule_id'], name: 'index_instructors_on_schedule_id'
   end
 
+  create_table 'room_blocks', force: :cascade do |t|
+    t.integer 'room_id', null: false
+    t.integer 'time_slot_id', null: false
+    t.boolean 'is_blocked', default: true
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['room_id'], name: 'index_room_blocks_on_room_id'
+    t.index ['time_slot_id'], name: 'index_room_blocks_on_time_slot_id'
+  end
+
```

Comment:
Where is the migration for this change?

Reply:
we decided to use room_bookings instead, because I didn't realize that we had the ability to block off times and rooms through that.